packages/cli/lang/en.lyaml

```diff
@@ -837,6 +837,20 @@ en:
           disableInitial: "Passing the \"--disable-initial\" option is no longer necessary. Running \"hs watch\" no longer uploads the watched directory by default."
           initialUpload: "To upload the directory run \"hs upload\" beforehand or add the \"--initial-upload\" option when running \"hs watch\"."
           notUploaded: "The \"hs watch\" command no longer uploads the watched directory when started. The directory \"{{ path }}\" was not uploaded."
+      preview:
+        describe: "Description of preview command"
+        positionals:
+          src:
+            describe: "Path to the local directory your files are in, relative to your current working directory"
+          dest:
+            describe: "Path in HubSpot Design Tools. Can be a net new path"
+        errors:
+          invalidPath: "The path \"{{ path }}\" is not a path to a directory"
+        options:
+          notify:
+            describe: "Log to specified file when a watch task is triggered and after workers have gone idle. Ex. --notify path/to/file"
+          initialUpload:
+            describe: "Initial upload to preview buffer (how can we better explain this to end user?)"
```

Comment:
```suggestion
            describe: "Upload directory before watching for updates"
```

I think we can keep close to the watch command here and keep it simple -- we don't have to expose information about the actual technical details to the developer on where their initial upload is going or what is happening behind the scenes